ethereum_bridge/src/parameters.rs

```diff
@@ -3,18 +3,41 @@ use std::num::NonZeroU64;
 
 use borsh::{BorshDeserialize, BorshSerialize};
 use eyre::{eyre, Result};
+use namada_core::ledger::eth_bridge::storage::whitelist;
 use namada_core::ledger::storage;
 use namada_core::ledger::storage::types::encode;
 use namada_core::ledger::storage::WlStorage;
 use namada_core::ledger::storage_api::{StorageRead, StorageWrite};
 use namada_core::types::ethereum_events::EthAddress;
 use namada_core::types::ethereum_structs;
 use namada_core::types::storage::Key;
+use namada_core::types::token::{Amount, Denomination};
 use serde::{Deserialize, Serialize};
 
 use crate::storage::eth_bridge_queries::{EthBridgeEnabled, EthBridgeStatus};
 use crate::{bridge_pool_vp, storage as bridge_storage, vp};
 
+/// An ERC20 token whitelist entry.
+#[derive(
+    Clone,
+    Copy,
+    Eq,
+    PartialEq,
+    Debug,
+    Deserialize,
+    Serialize,
+    BorshSerialize,
+    BorshDeserialize,
+)]
+pub struct Erc20WhitelistEntry {
```

Comment:
Do we also cap the max amount of wrapped nam that can exist on the bridge?

Reply:
oh uhhh good point. currently we don't. that check can also be added namada side

Reply:
I feel like that should be a separate PR though. It doesn't really concern the ERC20 token whitelist. The check should be made on the Bridge pool, to cap wNAM that is sent over to Ethereum.

Reply:
https://github.com/anoma/namada/issues/1722